packages/cart/src/models/cart.ts

```diff
@@ -47,18 +47,22 @@ export default class Cart {
   @Property({ columnType: "text" })
   currency_code: string
 
-  @OneToOne({
-    entity: () => Address,
-    joinColumn: "shipping_address_id",
-    cascade: [Cascade.REMOVE],
+  @Index({ name: "IDX_cart_shipping_address_id" })
+  @Property({ columnType: "text", nullable: true })
+  shipping_address_id?: string | null
+
+  @ManyToOne(() => Address, {
+    fieldName: "shipping_address_id",
     nullable: true,
   })
   shipping_address?: Address | null
 
-  @OneToOne({
-    entity: () => Address,
-    joinColumn: "billing_address_id",
-    cascade: [Cascade.REMOVE],
+  @Index({ name: "IDX_cart_billing_address_id" })
+  @Property({ columnType: "text", nullable: true })
+  billing_address_id?: string | null
```

Comment:
todo: update migration if not already done else where

Reply:
Migration has not been added yet